README.md

```diff
@@ -1,2 +1,60 @@
-# microverse-review-practice
-A project to practice reviewing codes from peers
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Code Review Practice
+
+> First Microverse Project
+
+This is a project that tests micronauts ability to review codes from their peers and other software engineers around the world. It also teaches how to do code review.
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Run you local server to run the app in your browser. For example, you can use visual studio code's live server to run the app**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### clone the app
+
+### Run 'npm install' from the command line
+
+### Run 'npx hint .' to test for html linting check
+
+### Run 'npx stylelint "**/*.{css,scss}"' to test for css linting check 
+
+
+
+## Authors
+
+👤 **Henry Kc**
+
+- GitHub: [@githubhandle](https://github.com/HENRYKC24)
+- Twitter: [@twitterhandle](https://twitter.com/henrykc24)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/henry-kc)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/HENRYKC24/microverse-review-practice/issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to Chibuzo Diala and Ruben Pire for the initial codes with which I used to pracitce this code review. I remain indebted to you.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
You could also add an empty line after line 60 here for the same reason as the previous.